base/math.jl

```diff
@@ -1070,6 +1081,35 @@ function frexp(x::T) where T<:IEEEFloat
     return reinterpret(T, xu), k
 end
 
+"""
+    $(@__MODULE__).scaleinv(x)
+
+Compute `(scale, invscale)` where `scale` and `invscale` are non-subnormal
+(https://en.wikipedia.org/wiki/Subnormal_number) finite powers of two such that
+`scale * invscale == 1` and `scale` is roughly on the order of `abs(x)`.
```

Comment:
Roughly on the order is a bit vague. Is there a more precise description?

Reply:
It's literally true that `eps(T) <= abs(x) * invscale < 4` except for when `x` is 0, Inf, or NaN (in which case the 0, Inf, or NaN is preserved by the finite `invscale`). However, for all `abs(x)` less than `inv(floatmin(T)) = prevfloat(floatmax(T)/4)` it holds that the product is `<2` and and for all `abs(x)` greater than `floatmin(T)` it holds that it's `>=1`.

I left the docstring vague to allow room to be flexible on the exact implementation and for future adjustments. But this could be documented as an unstable implementation detail.

Reply:
I added a little bit of info anyway, specifically documenting it as unstable bheavior.